runtimes/sklearn/mlserver_sklearn/sklearn.py

```diff
@@ -51,14 +52,27 @@ async def predict(self, payload: InferenceRequest) -> InferenceResponse:
 
     def _check_request(self, payload: InferenceRequest) -> InferenceRequest:
         if not payload.outputs:
+            found_predict_fn = False
+            if self.settings.parameters:
+                print(self.settings)
+                if self.settings.parameters.extra:
```

Comment:
Should we put it at the `parameters` level instead of `parameters.extra`?

Reply:
I suppose its following the definition of `extra` which is `Arbitrary settings, dependent on the inference runtime
    implementation.`

Reply:
Good point, let's leave it there.

I'm starting to see the `extra` field a bit redundant, as (semantically) `parameters` should already be able to hold extra model parameters. That's totally outside the scope of this PR though.